frontend/webpack/webpack.common.js

```diff
@@ -15,8 +15,8 @@ module.exports = {
   output: {
     publicPath: '/',
     path: resolve(__dirname, '../build'),
-    filename: '[name].bundle.js',
-    assetModuleFilename: 'images/[hash][ext][query]',
+    filename: '[name].[contenthash].js',
+    assetModuleFilename: 'images/[hash][ext]',
```

Comment:
`assetModuleFilename`도 `contenthash`로 바꾸는 것이 좋을 것 같습니다. 그리고 나중에 query를 사용할 수도 있으니 `[query]`는 굳이 제거하지 않아도 될 것 같아요.

```suggestion
    assetModuleFilename: 'images/[chunkhash][ext][query]',
```

https://github.com/woowacourse-teams/2022-moragora/issues/432#issuecomment-1250601889 를 참고해주세요.

Reply:
말씀해주신 `[query]`는 추후에 사용할 수 있으니 추가하는 것에 동의합니다! 
`[chunkhash]`, `[contenthash]` template string은 assets modules에 적용되지 않는 것을 확인했습니다.
<image width=200 src=https://user-images.githubusercontent.com/61308364/190964967-f52fd118-11bb-4031-98ce-5dd6ee765c75.png />
따라서 기존의 `[hash]`를 유지했습니다.
추가로 어떤 이미지가 ouput되었는지 명시하기 위해서 `[name]`을 추가했습니다.

결론은 `images/[name].[hash][ext][query]` 같이 설정하는 것으로 수정했습니다!